ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -291,6 +283,9 @@ objects:
         trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/setup-success; else touch /tmp/exit; fi; exit "${rc}"' EXIT
         trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
+        # We need jq later on
+        hash jq || exit 1
```

Comment:
Why is this needed?

Reply:
The installer creates a cluster with an example naming scheme of ci-op-ppc64le-8jslp-master-0.  I need to know the 8jslp part because the lb port is named with that name.  I grab it as follows:
CID=$(jq --raw-output .cluster_id /tmp/artifacts/installer/terraform.tfvars.json)
INGRESS_PORT="${CID}-ingress-port"
Since I already use a custom container with openshift-install, I also put the jq binary in there as well.  Do you know of another way to get it?

Reply:
Okay, I tried out JOB_NAME_HASH but that had the value of 477bf.  Whereas the cluster_id was nvdrv.

...
CLUSTER_NAME=ci-op-ppc64le
JOB_NAME_HASH=477bf
...
level=debug msg="2020-06-02T01:19:30.489Z [DEBUG] plugin.terraform-provider-openstack: 2020/06/02 01:19:30 [DEBUG] openstack_networking_port_v2 create options: openstack.PortCreateOpts{CreateOpts:ports.CreateOpts{NetworkID:\"c10be2e4-d19e-4b69-b896-b15ff2045d15\", Name:\"ci-op-ppc64le-nvdrv-ingress-port\", Description:\"\", AdminStateUp:(*bool)(0xc00160ad3c), MACAddress:\"\", FixedIPs:[]ports.IP{ports.IP{SubnetID:\"c529d3b4-c768-49b2-8498-3339f1d05319\", IPAddress:\"10.0.0.7\"}}, DeviceID:\"\", DeviceOwner:\"\", TenantID:\"\", ProjectID:\"\", SecurityGroups:(*[]string)(0xc0005b7520), AllowedAddressPairs:[]ports.AddressPair{}}, ValueSpecs:map[string]string{}}"
...

Reply:
So is `hash` just looking jq to see if it's present so you can use it to lookup the hash item that gets appended to the cluster name later?

If so, don't we control which image this is running - making this a redundant check? I mean, I see the value of short-circuiting if it's missing, but I guess I would think that we can guarantee that this check will never fail as long as we include it in the image being supplied.

Reply:
hash's purpose here is to check for the program and fail before a lot of work is done.  I figured that was far more preferable than failing deep in the bowels of code.  But I can remove that check.  Yes, we do own the image but it is possible that someone else could forget to add jq to a future container.

Reply:
I have no problem w/ this as is. :)